code/modules/client/preference/preferences_toggles.dm

```diff
@@ -224,10 +224,10 @@
 	set name = "Toggle Karma Gains"
 	set category = "Special Verbs"
 	set desc = "This button will allow you to stop other people giving you karma."
+	prefs.toggles ^= DISABLE_KARMA
+	prefs.save_preferences(src)
 	if (prefs.toggles & DISABLE_KARMA)
```

Comment:
spacing between if and bracket somehow passed review :(
Space needs to be removed from between `if` and `(....)